code/game/turfs/open.dm

```diff
@@ -119,8 +119,10 @@
 					step(C, olddir)
 					C.spin(1,1)
 		else if(lube&SLIDE_ICE)
+			C.slipping = TRUE
 			spawn(1)
 				step(C, olddir)
+			C.slipping = FALSE
```

Comment:
this literally should not work
you are spawning off the step() so, accordingly, you are setting this to TRUE and then just fucking setting it back to FALSE